### DISCUSSION TITLE
Welcome to RISC-OS-Community Discussions!

### DISCUSSION THREAD

**Discussion by pzaino** [RISC-OS-Community/RISC-OS-Community#9] [Announcements]
## 👋 Welcome!
  This is a place for public discussions about the RISC OS Community on GitHub projects, directions, future.

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 And thanks everyone for your incredible help and support :)

**Comment by FarbodNajafi11** [comment]
Does anyone know how to program a menu on LCD screen using RISC V ?

**Reply by skymandr** [reply]
This community is mostly concerned with RISC-OS, the operating system used for the original ARM computers, not with RISC-V.
